deluge/common.py

```diff
@@ -900,6 +909,33 @@ def free_space(path):
         return disk_data.f_bavail * block_size
 
 
+def is_interface(ip):
```

Comment:
The coma split is for multi-homing support while we did decide to focus on one thing in this PR I think it would be better kept as is so we don't have to re-write this again later.

Reply:
Sorry please remove and we'll create another function for that as needed